src/node_data.cpp

```diff
@@ -13,7 +13,10 @@
 namespace YAML {
 namespace detail {
 
-std::string node_data::empty_scalar;
+std::string& node_data::empty_scalar(){
```

Comment:
make sure to run clang-format; it would put a space before the {

Reply:
fixed

Reply:
fixed

Reply:
This line is for search engine:

undefined symbol: _ZN4YAML6detail9node_data12empty_scalarB5cxx11E

This change at least changed this much bug reports.

https://bugzilla.opensuse.org/show_bug.cgi?id=1178331
https://bugzilla.opensuse.org/show_bug.cgi?id=1160171
https://bugzilla.redhat.com/show_bug.cgi?id=1773357
https://github.com/calamares/calamares/issues/1312#issuecomment-587258287
https://aur.archlinux.org/packages/huggle/#comment-741229
https://stackoverflow.com/questions/52713571/mongod-symbol-lookup-error-mongod-undefined-symbol-zn4yaml6detail9node-data
https://bugzilla.opensuse.org/show_bug.cgi?id=1178332